build/test.ps1

```diff
@@ -125,8 +125,6 @@ if ($Env:ENABLE_PYTHON -eq "false") {
     Test-Python '../src/Python/qsharp-core' '../src/Python/qsharp-core/qsharp/tests'
 }
 
-Test-JavaScript '../src/Kernel'
```

Comment:
Should the function definition be removed as well as the call here? It looks like this was the only place `Test-JavaScript` was used.

Reply:
We could, but I left it in case we want to use it in the future.